api/openapi/v1/support-scheduler.yaml

```diff
@@ -5,36 +5,22 @@ info:
 servers:
 - url: http://localhost:48085/api
 paths:
-  /v1/info/{name}:
+  /v1/config:
     get:
-      description: Return Interval matching given name (interval names should be unique).
-        May be null if no interval matches on the name provided. Returns Internal
-        Service Error (HTTP 500) for unknown or unanticipated issues. NotFoundException
-        (HTTP 404) if no Interval is found with the provided name.
-      parameters:
-      - name: name
-        in: path
-        required: true
-        style: simple
-        explode: false
-        schema:
-          type: string
+      description: Fetch the current state of the service's configuration.
       responses:
         200:
-          description: interval matching on name
-          content:
-            '*/*':
-              schema:
-                $ref: '#/components/schemas/interval'
-        404:
-          description: if no interval is found for the identifier provided.
-        500:
-          description: for unknown or unanticipated issues
+          description: The service's configuration as JSON document
+        400:
```

Comment:
This endpoint also can return a `400 Bad Request` if the underlying configuration cannot be serialized to JSON properly.

Reply:
Let's add that to the description.